tests/system/Validation/CreditCardRulesTest.php

```diff
@@ -1220,32 +1205,26 @@ public function creditCardProvider()
     /**
      * Used to generate fake credit card numbers that will still pass the Luhn
      * check used to validate the card so we can be sure the cards are recognized correctly.
-     *
-     * @return string
      */
-    protected function generateCardNum(int $prefix, int $length)
+    private function generateCardNum(int $prefix, int $length): string
     {
         $pos        = mb_strlen($prefix);
         $finalDigit = 0;
         $sum        = 0;
 
-        // Fill in the first values of the string based on $prefix
         $string = str_split($prefix);
```

Comment:
The `$string` is an array. `$digits` is better?

Reply:
Done